packages/boost/BUILD

```diff
@@ -12,24 +12,25 @@ def build(ctx):
     with srcdir.path('project-config.jam').open('w') as f:
         f.write('using gcc : cloudabi : %s ;\n' % ctx.cxx())
     stage = srcdir.path('stage')
-    srcdir.run([
-        './b2',
-        '-j8',
-        '-d+2',
-        'binary-format=elf',
-        'link=static',
-        'toolset=gcc-cloudabi',
-        'cxxflags=' + ctx.cxxflags(),
-        '--without-python',
-        '--prefix=%s' % stage,
-        'install',
-    ] + {
-        'armv6': ['abi=aapcs', 'address-model=32', 'architecture=arm'],
-        'armv7': ['abi=aapcs', 'address-model=32', 'architecture=arm'],
-        'aarch64': ['abi=aapcs', 'address-model=64', 'architecture=arm'],
-        'i686': ['abi=sysv', 'address-model=32', 'architecture=x86'],
-        'x86_64': ['abi=sysv', 'address-model=64', 'architecture=x86'],
-    }[ctx.cpu()])
+    srcdir.run(
```

Comment:
It looks like you're use YAPF 0.20, whereas I was still using 0.19. Just upgraded YAPF on my system and got the same output. Fair enough!